test/cets_SUITE.erl

```diff
@@ -381,6 +383,19 @@ insert_new_fails_if_the_local_server_is_dead(_Config) ->
         exit:{noproc, {gen_server, call, _}} -> ok
     end.
 
+insert_serial_works(Config) ->
+    #{pid1 := Pid1, tab1 := Tab1, tab2 := Tab2} = given_two_joined_tables(Config),
+    ok = cets:insert_serial(Pid1, {a, 1}),
+    [{a, 1}] = cets:dump(Tab1),
+    [{a, 1}] = cets:dump(Tab2).
+
+insert_serial_overwrites_data(Config) ->
```

Comment:
I don't see the point of this test, because if I replace `insert_serial` with just `insert`, the test would also pass. IMO it should check that `insert_serial` is in some way different than just `insert`.

Reply:
good, added three tests to show difference between insert_new, insert and insert_serial